client/src/hooks/location-types/index.ts

```diff
@@ -0,0 +1,56 @@
+import { useMemo } from 'react';
+import { useQuery, UseQueryResult, UseQueryOptions } from 'react-query';
+import { apiRawService } from 'services/api';
+
+const DEFAULT_QUERY_OPTIONS: UseQueryOptions = {
+  placeholderData: [],
+  retry: false,
+  keepPreviousData: true,
+  refetchOnWindowFocus: false,
+};
+
+export type LocationTypesParams = {
+  materialIds?: string[];
+  supplierIds?: string[];
+  businessUnitIds?: string[];
+  originIds?: string[];
+};
+
+export type LocationType = {
+  label: string;
+  value: 'point-of-production' | 'aggregation-point' | 'country-of-production' | 'unknown';
+};
+
+type ResponseData = UseQueryResult<LocationType[]>;
+
+export function useLocationTypes(
+  params: LocationTypesParams,
+  options: UseQueryOptions = {},
+): ResponseData {
+  const query = useQuery(
+    ['location-types', JSON.stringify(params)],
+    () =>
+      apiRawService
```

Comment:
```suggestion
    () =>
     async apiRawService
```

Reply:
In the documentation and examples the async is there, I'm not sure if it's needed because it seems working well but I'll recommend to add it just to keep a consistency with the other hooks